src/autoscaler/metricsforwarder/cmd/metricsforwarder/metricsforwarder_suite_test.go

```diff
@@ -109,6 +109,10 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	cfg.Logging.Level = "debug"
 	cfg.MetronAddress = grpcIngressTestServer.GetAddr()
 	cfg.Server.Port = 10000 + GinkgoParallelNode()
+	cfg.Server.NodeAddrs = []string{"localhost"}
```

Comment:
I think sharing is not necessary for metric forwarder

Reply:
Yes, sharding has been removed.